packages/fluent-editor/src/config/editor.utils.ts

```
@@ -168,14 +168,14 @@ export function getEventComposedPath(evt) {
   return path
 }
 
-export const sanitize = (url, protocols) => {
+export function sanitize(url, protocols) {
   const anchor = document.createElement('a')
   anchor.href = url
   const protocol = anchor.href.slice(0, anchor.href.indexOf(':'))
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)